kloppy/tests/test_opta.py

```diff
@@ -59,3 +61,20 @@ def test_correct_deserialization(self):
             end_timestamp=1537721737.788,
             attacking_direction=AttackingDirection.NOT_SET,
         )
+
+        assert dataset.events[0].coordinates == Point(50.1, 49.4)
+
+    def test_correct_normalized_deserialization(self):
+        base_dir = os.path.dirname(__file__)
+
+        serializer = OptaSerializer()
+
+        with open(f"{base_dir}/files/opta_f24.xml", "rb") as f24_data, open(
+            f"{base_dir}/files/opta_f7.xml", "rb"
+        ) as f7_data:
+
+            dataset = serializer.deserialize(
+                inputs={"f24_data": f24_data, "f7_data": f7_data}
+            )
+
+        assert dataset.events[0].coordinates == Point(0.501, 0.506)
```

Comment:
Opta is 100x100. That does mean normalised to 1x1 should result in `0.501, 0.494`, right?

Reply:
oh. see my comment about `vertical_orientation` of KloppyCoordinateSystem